src/mpid/ch4/netmod/ofi/init_settings.c

```diff
@@ -215,6 +215,14 @@ int MPIDI_OFI_init_hints(struct fi_info *hints)
         /*     PROTOCOL         |  CONTEXT  |        SOURCE         |       TAG          */
         MPIDI_OFI_PROTOCOL_MASK | 0 | MPIDI_OFI_SOURCE_MASK | 0 /* With source bits */ :
         MPIDI_OFI_PROTOCOL_MASK | 0 | 0 | MPIDI_OFI_TAG_MASK /* No source bits */ ;
+
+    /* should we check MPIR_CVAR_ENABLE_GPU? */
```

Comment:
I think yes, we should check the CVAR before enabling.

Reply:
Updated.